docroot/themes/humsci/humsci_basic/src/scss/components/_global-footer.scss

```diff
@@ -0,0 +1,13 @@
+// These styles restore brand styles imported from Decanter that were overwritten by humsci_basic theme base styles.
+// Decanter Global Footer: https://decanter.stanford.edu/component/identity-global-footer/
+.hb-global-footer {
+  .su-logo span {
```

Comment:
oh @heatherdesigns were you going to change this to .fa-ext like kasey's styles?